code/modules/projectiles/guns/ballistic.dm

```diff
@@ -72,6 +72,8 @@
 	var/casing_ejector = TRUE
 	///Whether the gun has an internal magazine or a detatchable one. Overridden by BOLT_TYPE_NO_BOLT.
 	var/internal_magazine = FALSE
+	///Should we attempt to display the gun's magazine?
+	var/display_mag = TRUE
```

Comment:
is it really necessary add an addition check for **_every single time a gun is fired_** when only a single meme weapon utilizes it? why not just add an invisible icon for the stickman gun instead?

Reply:
if I'm honest I thought I'd find more like this, I'll pull the var and add a invisible icon

Reply:
wait a sec this  var already exists, it's just totally unused. wtf